### PR TITLE
added a space to user controller

### PR DESCRIPTION
**Pull request by drewvpham**

### CODE REVIEW

**Review by marshall-alex — APPROVED** [review]
thats a dope space homie, you got my approval dawg

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,2 +1,13 @@
 class UsersController < ApplicationController
+
```
Comment (bribenn) [inline app/controllers/users_controller.rb:2]: why did you add a space?!

**Review by bribenn — APPROVED** [review]
